### ISSUE TITLE
Questions + figure regarding how it works

### ISSUE THREAD

**Issue by Ezwen** [anewusername/mxpp#2]
Hi! I find your project really interesting for potentially helping with my migration to matrix in the long run :). I have two questions regarding the features that you aim to provide:
- Do you plan to support MUC and/or MUC 1-1 conversations as well?
- I tried to make a figure of what your tool does, did I get it right? (see below)

![attempt-bridge2](https://cloud.githubusercontent.com/assets/1388961/22158404/583b4f14-df3c-11e6-944f-1c84031d3ec9.png)

and thanks for your work! I'll probably try it soon.

**Comment by anewusername** [comment]
Yep, that figure is correct -- glad to hear that the project might be useful for you!

I don't personally use MUCs, but I'll certainly take a look at implementing them next.

**Comment by anewusername** [comment]
I've implemented basic MUC support.
You have to join/leave manually with text commands since I haven't found a way to reliably test invitations, but messaging seems to work fine.
